### PR TITLE
LPS-164603 Poshi Automation | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-164603

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase [hunk 1]
```diff
@@ -1430,4 +1430,48 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-164269. The navigation menu creator could view a tooltip with the page URL on selection modal."
+	@priority = "4"
+	test ViewPageUrlTooltip {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase:1435]: ViewPageURLOnTooltipWhenSelectPageItem

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenusWithStaging.testcase [hunk 2]
```diff
@@ -1450,4 +1450,50 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-164269. The navigation menu creator could view a tooltip with the page URL on selection modal on staging site."
+	@priority = "4"
+	test ViewPageUrlTooltipOnStagingSite {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenusWithStaging.testcase:1455]: ViewPageURLOnTooltipWhenSelectPageItem

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithMapping.testcase [hunk 3]
```diff
@@ -1939,4 +1939,80 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-164269. The page creator could view a tooltip with the page URL when mapping a fragment on sites and libraries."
+	@priority = "4"
+	test ViewPageUrlTooltipOnLinkToPage {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithMapping.testcase:1944]: ViewPageURLOnTooltipWhenSelectPageItem

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithMapping.testcase [hunk 4]
```diff
@@ -1939,4 +1939,80 @@ definition {
...
+		task ("Given a page creator maps a Button fragment to Page on Link fied") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithMapping.testcase:1945]: Given a page creator opens the link to page modal

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithMapping.testcase [hunk 5]
```diff
@@ -1939,4 +1939,80 @@ definition {
...
+					key_text = "plus",
+					locator1 = "Icon#ANY_ITEM_SELECTOR");
+
+				SelectFrame(locator1 = "IFrame#MODAL_BODY");
+
+				Portlet.expandTree();
+			}
+		}
+
+		task ("When the page creator mouses hover the content page") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithMapping.testcase:1985]: When the page creator hovers on the content page item

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase [hunk 6]
```diff
@@ -1430,4 +1430,48 @@ definition {
...
+		task ("Given a navigation menu creator opens the select pages modal of navigation menu") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenus.testcase:1436]: Given a navigation menu opens the select pages modal

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenusWithStaging.testcase [hunk 7]
```diff
@@ -1450,4 +1450,50 @@ definition {
...
+		task ("Given a navigation menu creator opens the select pages modal of navigation menu in staging site") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/sitenavigation/navigationmenu/NavigationMenusWithStaging.testcase:1456]: Given a navigation menu opens the select pages modal

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind updating according to comments? Thanks

**Comment by ozysouza** [comment]
> Hi @ozysouza, would you mind updating according to comments? Thanks

Hi @Tim-Cao, PR updated. Could you please have a look at? Thanks!

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**